src/us/kbase/workspace/test/database/provenance/FundingReferenceTest.java

```diff
@@ -189,64 +158,48 @@ private void buildFundingRefFailWithError(final FundingReference.Builder builder
 	}
 
 	@Test
-	public void buildFailInvalidPIDs() throws Exception {
-		for (String invalidPid : INVALID_PID_LIST) {
-			buildFundingRefFailWithError(
-					FundingReference.getBuilder(FUNDER_NAME_STRING).withFunderID(invalidPid),
-					"Illegal format for funderID: \"" + invalidPid + "\"\n" +
-							"It should match the pattern \"^([a-zA-Z0-9][a-zA-Z0-9\\.]+)\\s*:\\s*(\\S.+)$\"");
-		}
-	}
-
-	@Test
-	public void buildFailInvalidAwardURL() throws Exception {
+	public void buildFailInvalidGrantURL() throws Exception {
 		buildFundingRefFailWithError(
-				FundingReference.getBuilder(FUNDER_NAME_STRING).withAwardURL(FUNDER_ID_STRING),
-				"Illegal awardURL url '" + FUNDER_ID_STRING + "': unknown protocol: ror");
+				FundingReference.getBuilder(ORG_1).withGrantURL(GRANT_ID_STRING),
+				"Illegal grantURL url '" + GRANT_ID_STRING + "': no protocol: " + GRANT_ID_STRING);
```

Comment:
This test now seems to duplicate the test below, as they both test the no protocol case rather than one testing the incorrect protocol case

Reply:
Fixed to test a different aspect of the URL

Reply:
:+1: